tests/Game.class.test.js

```diff
@@ -0,0 +1,965 @@
+/* eslint-disable max-len */
+'use strict';
+
+const Game = require('../src/modules/Game.class');
+
+function sum(game) {
+  let result = 0;
+
+  for (const row of game.getState()) {
+    for (const n of row) {
+      result += n;
+    }
+  }
+
+  return result;
+}
+
+function count(game) {
+  return game.getState().flat().filter(n => n).length;
+}
+
+function copyState(state) {
+  return state.map(row => [...row]);
+}
+
+function transposeState(state) {
+  const result = [];
+
+  for (let col = 0; col < 4; col++) {
+    result[col] = [];
+
+    for (let row = 0; row < 4; row++) {
+      result[col].push(state[row][col]);
+    }
+  }
+
+  return result;
+}
+
+describe('Game', () => {
+  describe('', () => {
+    it('should have a constructor', () => {
+      expect(typeof Game).toBe('function');
+      expect(typeof Game.prototype.constructor).toBe('function');
+    });
+
+    [
+      'getState',
+      'getScore',
+      'getStatus',
+      'moveLeft',
+      'moveRight',
+      'moveUp',
+      'moveDown',
+      'start',
+      'restart',
+    ].forEach((method) => {
+      it(`should have a "${method}" method`, () => {
+        const game2048 = new Game();
+
+        expect(typeof game2048[method]).toBe('function');
+      });
+    });
```

Comment:
The first `describe` block has an empty description string. It's best practice to provide a meaningful description for each test suite for better readability and understanding of the test cases.